alacritty/src/renderer/box_drawing.rs

```diff
@@ -0,0 +1,584 @@
+                | '\u{2520}' | '\u{2521}' | '\u{2523}' | '\u{2526}' | '\u{2528}' | '\u{2529}'
+                | '\u{252b}' | '\u{2538}' | '\u{2539}' | '\u{253a}' | '\u{253b}' | '\u{2540}'
+                | '\u{2542}' | '\u{2543}' | '\u{2544}' | '\u{2547}' | '\u{2549}' | '\u{254a}'
+                | '\u{254b}' | '\u{2579}' | '\u{257f}' => heavy_stroke_size,
+                _ => 0,
+            };
+            // Bottom vertical line.
+            let stroke_size_v2 = match character {
+                '\u{2502}' | '\u{250c}' | '\u{250d}' | '\u{2510}' | '\u{2511}' | '\u{251c}'
+                | '\u{251d}' | '\u{251e}' | '\u{2521}' | '\u{2524}' | '\u{2525}' | '\u{2526}'
+                | '\u{2529}' | '\u{252c}' | '\u{252d}' | '\u{252e}' | '\u{252f}' | '\u{253c}'
+                | '\u{253d}' | '\u{253e}' | '\u{253f}' | '\u{2540}' | '\u{2543}' | '\u{2544}'
+                | '\u{2547}' | '\u{2577}' | '\u{257f}' => stroke_size,
+                '\u{2503}' | '\u{250e}' | '\u{250f}' | '\u{2512}' | '\u{2513}' | '\u{251f}'
+                | '\u{2520}' | '\u{2522}' | '\u{2523}' | '\u{2527}' | '\u{2528}' | '\u{252a}'
+                | '\u{252b}' | '\u{2530}' | '\u{2531}' | '\u{2532}' | '\u{2533}' | '\u{2541}'
+                | '\u{2542}' | '\u{2545}' | '\u{2546}' | '\u{2548}' | '\u{2549}' | '\u{254a}'
+                | '\u{254b}' | '\u{257b}' | '\u{257d}' => heavy_stroke_size,
+                _ => 0,
+            };
+
+            let x_v = canvas.center_h();
+            let y_h = canvas.center_v();
+
+            let v_line_bounds_top = canvas.v_line_bounds(x_v, stroke_size_v1);
+            let v_line_bounds_bot = canvas.v_line_bounds(x_v, stroke_size_v2);
+            let h_line_bounds_left = canvas.h_line_bounds(y_h, stroke_size_h1);
+            let h_line_bounds_right = canvas.h_line_bounds(y_h, stroke_size_h2);
+
+            let size_h1 = max(v_line_bounds_top.1 as i32, v_line_bounds_bot.1 as i32) as f32;
+            let x_h = min(v_line_bounds_top.0 as i32, v_line_bounds_bot.0 as i32) as f32;
+            let size_h2 = width as f32 - x_h;
+
+            let size_v1 = max(h_line_bounds_left.1 as i32, h_line_bounds_right.1 as i32) as f32;
+            let y_v = min(h_line_bounds_left.0 as i32, h_line_bounds_right.0 as i32) as f32;
+            let size_v2 = height as f32 - y_v;
+
+            // Left horizontal line.
+            canvas.draw_h_line(0., y_h, size_h1, stroke_size_h1);
+            // Right horizontal line.
+            canvas.draw_h_line(x_h, y_h, size_h2, stroke_size_h2);
+            // Top vertical line.
+            canvas.draw_v_line(x_v, 0., size_v1, stroke_size_v1);
+            // Bottom vertical line.
+            canvas.draw_v_line(x_v, y_v, size_v2, stroke_size_v2);
+        },
+        // Light and double line box components:
+        // '═','║','╒','╓','╔','╕','╖','╗','╘','╙','╚','╛','╜','╝','╞','╟','╠','╡','╢','╣','╤','╥',
+        // '╦','╧','╨','╩','╪','╫','╬'.
```

Comment:
I'd assume the "light" part here was a copy/paste error? These are only double-line box components.

Reply:
Nah, look on the unicode URL, and if you look closely some of them have single light lines in them.

Reply:
But even the ones with the light lines don't have light line in the name? It's always "single", never "light"?